src/Tests/Microsoft.NET.Build.Containers.IntegrationTests/ContainerCli.cs

```diff
@@ -34,7 +34,10 @@ public static RunExeCommand InspectCommand(ITestOutputHelper log, params string[
     public static RunExeCommand LoadCommand(ITestOutputHelper log, params string[] args)
       => CreateCommand(log, "load", args);
 
-    private static RunExeCommand CreateCommand(ITestOutputHelper log, string command, string[] args)
+    public static RunExeCommand PortCommand(ITestOutputHelper log, string containerName, int port) =>
```

Comment:
The test that was failing was often failing due to colliding port numbers - I've fixed this by letting Docker assign host ports, then parsing those ports using this command. This makes us much less likely to fail.